timer/timer.ino

```diff
@@ -15,6 +15,8 @@ Adafruit_SSD1306 display(128, 64, &Wire, -1);
 SoftwareSerial mySerial(D5, D6);
 Timer t;
 
+// set to true/false when using another type of reed sensor
+bool reedOpenSensor = false;
```

Comment:
I would leave this to true, since most people will have a normal open sensor. You are the first person to use a NC sensor.

Reply:
Just changed the default value to `true`.